packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss

```diff
@@ -103,7 +103,7 @@ $ms-DocumentCardActions-verticalPadding: 4px;
 
   :global(.ms-Button:hover) {
     :global(.ms-Button-icon) {
-      color: #1174c3;
+      color: $ms-color-neutralLight;
```

Comment:
prefer to use one of the button semantic slots

Reply:
Which one?